cmake/ecbuild_generate_fortran_interfaces.cmake

```diff
@@ -69,7 +69,24 @@
 
 function( ecbuild_generate_fortran_interfaces )
 
-  find_program( FCM_EXECUTABLE fcm REQUIRED DOC "Fortran interface generator" )
+  find_program( FCM_EXECUTABLE fcm DOC "Fortran interface generator"
+                HINTS
+                  ${CMAKE_SOURCE_DIR}/fcm
+                  ${CMAKE_BINARY_DIR}/fcm
+                  ${fcm_ROOT}
+                  ENV fcm_ROOT
+                  PATH_SUFFIXES bin )
+  if (NOT FCM_EXECUTABLE)
+    include(FetchContent)
+    FetchContent_Populate(
+      fcm
+      URL            https://github.com/metomi/fcm/archive/refs/tags/2019.09.0.tar.gz
```

Comment:
maybe make the version (or better the file name) to download a variable so the user can override? make 2019.09.0 the default.

Reply:
Done. Thanks for the recommendation.